ister.py

```diff
@@ -68,7 +68,9 @@
 from urllib.parse import urlparse
 from contextlib import closing
 import netifaces
-import pycryptsetup
+import gi
+gi.require_version("BlockDev", "2.0")
+from gi.repository import BlockDev as bd
```

Comment:
Oh fun, gobject introspection. Well I guess that's making good use of the gnome stack.